sdk/web-pubsub/web-pubsub-express/src/cloudEventsDispatcher.ts

```diff
@@ -225,14 +260,51 @@ export class CloudEventsDispatcher {
     return value[0];
   }
 
-  private GetContext(ce: CloudEvent, origin: string): ConnectionContext {
+  /**
+   * Convert the states object to base64 JSON string
+   * @param states the connection states
+   * @returns The base64 JSON string
+   */
+  private stringifyStates(states: Record<string, string>): string {
```

Comment:
A lot of these private helpers don't seem to be methods at all (there is no reference to `this`) so why place them on the class as privates instead of leaving them as functions in module scope?

Reply:
Is it a coding convention or performance consideration involved? Is it that when there is no `this` we tend to put the method to a `util` module?

Reply:
it doesn't have to be in a separate module, but something like this:

```ts
function helper() { 
 /* do something useful */
}

export class SomeClass {
   public someMethod() {
       return helper();
   }
}
```

There are a few reasons I prefer this to private methods:

1. TypeScript private methods aren't actually private, you can see them and call them in JS at runtime
2. TypeScript considers private methods as part of your type, so if you have two class instances that differ only in private members, they are not type compatible.

Reply:
Thanks for the explaination!